main.go

```diff
@@ -918,11 +921,16 @@ func writePidFile(loadedConfig *config.GatewayConfig, logger log.Logger) {
 		if err := ioutil.WriteFile(*loadedConfig.PidFilename, []byte(strconv.FormatInt(int64(pid), 10)), os.FileMode(0644)); err != nil {
 			logger.Log(log.Err, err, logkey.Filename, *loadedConfig.PidFilename, "cannot store pid in pid file")
```

Comment:
Should we downgrade this to a warning? People seem to get stuck up on it in their logs when that's failing.

Reply:
The logger has Msg or Err as the log levels, so there isn't a "warning" level. @mdubbyap mentioned that Err is the appropriate level (since this is an error, and anything fatal would just cause the gateway to shutdown). But I can switch this to Msg if we think that will be less confusing for users

Reply:
Nevermind then.